### PR TITLE
React redux by m rozenek

### PR DESCRIPTION
**Pull request by DawidRozenek**

### CODE REVIEW

File: content/posts/2018-04-19--how-to-implement-redux-in-react/index.md [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+
+`subscribe` - let you subscribe on store changes and register a callback
+
+Store shouldn't be mutated!
+You should always return a new object.
+To avoild an Object mutations, you can use those methods:
+
+- spread operator:
+```js
+return {
```
Comment (MichalCiesliczka) [inline content/posts/2018-04-19--how-to-implement-redux-in-react/index.md]: Something is wrong with indentation on preview here

File: content/posts/2018-04-19--how-to-implement-redux-in-react/index.md [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+
+`getState` - return current state
+
+`dispatch` - let you dispatch actions
+
+`subscribe` - let you subscribe on store changes and register a callback
+
+Store shouldn't be mutated!
+You should always return a new object.
+To avoild an Object mutations, you can use those methods:
```
Comment (MichalCiesliczka) [inline content/posts/2018-04-19--how-to-implement-redux-in-react/index.md]: Maybe add some small info why one is better or when I should use this or that

File: content/posts/2018-04-19--how-to-implement-redux-in-react/index.md [hunk 3]
```diff
@@ -0,0 +1,869 @@
+---
```
Comment (MichalCiesliczka) [inline content/posts/2018-04-19--how-to-implement-redux-in-react/index.md]: That's daaamn long. Maybe you should think about splitting this into parts?

File: content/posts/2018-04-19--how-to-implement-redux-in-react/index.md (hunk 1 above)
Comment (mro-agnitio) [reply content/posts/2018-04-19--how-to-implement-redux-in-react/index.md]: I don't know why :)
I tried to fix it... but it don't want to listen to me :smile:

**Pull request merged**